### PR TITLE
Jrh master label

### PR DESCRIPTION
**Pull request by JanRuneHenriksen**

### CODE REVIEW

File: src/components/Inputs/Label/Label.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import { Label as UIFabLabel } from 'office-ui-fabric-react/lib/Label';
+
+/** Label example */
+const label ="I am a label";
+function Label({required, disabled }) {
```
Comment (nwinger) [inline src/components/Inputs/Label/Label.js]: Being a label, you should probably accept a 'label' prop to the component? Current example will always display "I am a label".

File: src/components/Inputs/Label/Label.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	return (<UIFabLabel required={required} disabled={disabled} >{label}</UIFabLabel>);
```
Comment (nwinger) [inline src/components/Inputs/Label/Label.js]: What purpose does required and disabled serve for a label component?

File: src/components/Inputs/Label/Label.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+}
+
+Label.propTypes = {
```
Comment (nwinger) [inline src/components/Inputs/Label/Label.js]: Add the label property as a required propType of type string.

File: t [hunk 4]
```diff
@@ -0,0 +1,79 @@
+[33m1d85ba5[m[33m ([m[1;36mHEAD -> [m[1;32mjrh_master_10_04_18_no_text[m[33m)[m - Added ariaDescription to button - Modified checkbox disabled label text
```
Comment (nwinger) [inline t]: What is this file? Seems to contain git logs.

File: src/components/Inputs/Label/Label.js (hunk 1 above)
Comment (JanRuneHenriksen) [reply src/components/Inputs/Label/Label.js]: Yeah

File: src/components/Inputs/Label/Label.js (hunk 3 above)
Comment (JanRuneHenriksen) [reply src/components/Inputs/Label/Label.js]: Yeah

File: t (hunk 4 above)
Comment (JanRuneHenriksen) [reply t]: Hmm... mystery. Will delete!

**Comment by nwinger** [comment]
Closing this one.

**Pull request closed without merging**